src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstValue = Big(numberOne);
+  const secondvalue = Big(numberTwo);
+
+  if (operation === '-') {
+    return firstValue - secondvalue;
+  }
+
+  if (operation === '+') {
+    return firstValue + secondvalue;
+  }
+
+  if (operation === 'x') {
+    return firstValue * secondvalue;
+  }
+
+  if (operation === '/') {
+    if (parseInt(secondvalue, 10) === 0) { return "You can't divide a number by zero"; }
+    return firstValue / secondvalue;
+  }
```

Comment:
- [ ] Please use the Big.js math operators(plus, minus, times, div and modifier), not the regular math operations. Eg: `firstValue.div(secondValue)`

- [ ] In addition, please include a case for the percentage operator(%) dividing the number by 100.

Reply:
Done.